DonjinKrawler-Client/src/main/java/donjinkrawler/Client.java

```diff
@@ -58,7 +58,10 @@ public void received(Connection connection, Object object) {
                     game.changeShellPosition(msg);
                 } else if (object instanceof EnemyPacket && game != null) {
                     EnemyPacket enemyPacket = (EnemyPacket) object;
-                    game.addEnemies(enemyPacket.getEnemies());
+                    if (enemyPacket.isUpdate())
```

Comment:
Lots of checkstyle violations on this commit, please check before merging. Lemme know if you need help setting that up in IntelliJ as well. Not gonna point all of them out, but running checkstyle and / or code formatting should speed cleaning up the violations.